conf/routes

```diff
@@ -11,6 +11,8 @@ POST        /ui/supersede/:hash                              @controllers.html.U
 GET         /load                                            @controllers.api.ImportData.loadWithProgress()
 GET         /load/progress                                   @controllers.api.ImportData.progress()
 
+GET         /all                                             @controllers.api.Rest.all()
```

Comment:
Why do we need another api for this? can we do a search and read the limit from the query parameter?
http://school.openregister.org/search?_query=&_limit=10000000